html/pages/preferences.inc.php

```diff
@@ -173,6 +177,42 @@
     }//end if
 }//end if
 
+echo "<h3>Default Dashboard</h3>
+<hr>
+<div class='well'>";
+echo $updatedashboard_message;
+echo "
+  <form method='post' action='preferences/' class='form-horizontal' role='form'>
+    <div class='form-group'>
+      <input type=hidden name='action' value='changedash'>
+      <div class='form-group'>
+        <label for='dashboard' class='col-sm-2 control-label'>Dashboard</label>
+        <div class='col-sm-4'>
+          <select class='form-control' name='dashboard'>";
+$defdash = dbFetchCell("SELECT dashboard FROM users WHERE user_id = ?", array($_SESSION['user_id']));
+$dashoptions = dbFetchRows(
+    "SELECT dashboards.*,users.username
+    FROM `dashboards`
+    INNER JOIN `users` ON users.user_id = dashboards.user_id
+    WHERE (dashboards.access > 0 && dashboards.user_id != ?) || dashboards.user_id = ?",
+    array($_SESSION['user_id'],$_SESSION['user_id'])
+);
+
+foreach ($dashoptions as $dash) {
+    echo "
+            <option value='".$dash['dashboard_id']."'".($defdash == $dash['dashboard_id'] ? ' selected' : '').">".$dash['username'].':'.$dash['dashboard_name']."</option>";
```

Comment:
please wrap `$dash['username']` and `$dash['dashboard_name']` in display() function